packages/editor/src/components/collab-sidebar/commentButtonToolbar.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton } from '@wordpress/components';
+import { _x } from '@wordpress/i18n';
+import { comment as commentIcon } from '@wordpress/icons';
+import { __unstableCommentIconToolbarFill as CommentIconToolbarFill } from '@wordpress/block-editor';
+
+const AddCommentToolbarButton = ( { onClick } ) => {
+	return (
+		<CommentIconToolbarFill>
+			<ToolbarButton
+				accessibleWhenDisabled
+				icon={ commentIcon }
+				label={ _x( 'Comment', 'Open comment button' ) }
```

Comment:
Is there any difference between "Add comment button" and "Open comment button"? Should we just use `noun` or `verb` as the context in all cases where we have the `Comment` string?

Reply:
Yes, the "Open comment button" icon is displayed when a block has an existing comment. Clicking this icon opens the sidebar, allowing users to view comments. In contrast, the "Add comment button" icon enables users to add a new comment to a block.